src/garage/experiment/local_tf_runner.py

```diff
@@ -182,7 +193,7 @@ def obtain_samples(self, itr, batch_size):
             logger.log('Obtaining samples...')
         return self.sampler.obtain_samples(itr, batch_size)
 
-    def save(self, epoch, paths=None):
+    def _save(self, epoch, paths=None):
```

Comment:
i recommend you rename this to `_maybe_save` since this is not guaranteed to save a snapshot.

do we really want to never save a snapshot? is this a use case we want to support? it seems like the code would be simpler if no snapshots was an error.

@krzentner WDYT?